<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelThreads">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxThreading">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Threading</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeSettings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequenz = </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
